package.json

```diff
@@ -36,11 +36,13 @@
     "test": "mocha test/unit --recursive && mocha test/runner --recursive"
   },
   "dependencies": {
+    "allure-js-commons": "^1.3.2",
```

Comment:
Looks like this dep should be in dev dependencies. If I don't need allure repots, I don't want to install it with codeceptjs

Reply:
It's a very basic package. It doesn't install Allure by itself, only some JS APIs.
I'd rather keep it in package.json to simplify Allure installation. 

In future versions, this will probably be changed.